tools/luh2/luh2.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+# LUH2 python script
+# Usage: python luh2.py -l <luh2file> -s <luh2staticfile>
+
```

Comment:
Usage should match the README, right?
python luh2.py -l <raw-luh2-datafile> -s <luh2-static-datafile> -r <regrid-targetfile> -w <regridder-output> -o <outputfile>

Reply:
Good catch.  Resolved per  9cbfe4f0a10196e7805aec032205746b455eb92b